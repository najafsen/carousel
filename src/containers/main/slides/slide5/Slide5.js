import React from "react";
import "./Slide5.scss";

export const Slide5 = () => (
  <div className="slide-content">
    <div className="slide5">
      <h1>Or the view from my window in the morning! :)</h1>
    </div>
  </div>
);
