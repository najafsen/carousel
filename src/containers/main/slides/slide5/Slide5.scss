.slide5 {
  position: relative;
  display: flex;
  flex: 1;
  padding: 16px 96px;
  background-image: url("./img/my-morning-view.jpg");
  background-size: cover;

  h1 {
    position: absolute;
    width: 400px;
    top: 244px;
    color: white;
    font-size: 32px;
  }
}
