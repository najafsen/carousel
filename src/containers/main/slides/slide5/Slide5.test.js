import React from "react";
import { render } from "@testing-library/react";
import { Slide5 } from "./Slide5";

describe("Slide5", () => {
  test("matches snapshot", () => {
    const { asFragment } = render(<Slide5 />);
    expect(asFragment()).toMatchSnapshot();
  });
});
