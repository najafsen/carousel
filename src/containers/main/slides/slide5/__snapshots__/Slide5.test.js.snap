// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Slide5 matches snapshot 1`] = `
<DocumentFragment>
  <div
    class="slide-content"
  >
    <div
      class="slide5"
    >
      <h1>
        Or the view from my window in the morning! :)
      </h1>
    </div>
  </div>
</DocumentFragment>
`;
